### PR TITLE
Fields for PID and rate profile names.

### LINKED ISSUE #3064: PID Profile Names
### Is your feature request related to a problem? Please describe

I carry a range of different payloads on my cinelifter and 5" aircrafts at different altitudes. I establish different PID tunes for the different payload types that I carry. Depending on what I payload I need to fly, I select different PID profiles to match my situation.

It would be really helpful if each PID Profile in the configurator could be nick-named so that you wouldn't have to remember or make external note of what each PID profile corresponded to. 

In addition to that, if there was any more room to store more PID profiles, like maybe 6 instead of 3, that'd be great too.

### Describe the solution you'd like

Allow pid profiles to be given alphanumeric nicknames in the configurator.

### Describe alternatives you've considered

A note in my phone to remember what PID profile is for what payload profile

### Other information

_No response_

### PR DESCRIPTION
**Pull request by qvasic**
Fixes #3064

Some pilots have different PID profiles for different payloads/conditions they fly(lighter/bigger battery, action camera/no camera, various elevations), different rate profiles for different styles of flying (cine, race, freestyle).
Having readable profile names would significantly simplify selecting the proper profile, as opposed to memorizing profile numbers - just imagine having N quads with M profiles each.

image

image

### CODE REVIEW

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:86]:
```suggestion
        } else {
```

File: src/js/tabs/pid_tuning.js
Comment (haslinghuis) [inline src/js/tabs/pid_tuning.js:2256]: Space (no tabs) / indentation of 4 spaces please.

File: src/js/msp/MSPHelper.js
Comment (haslinghuis) [inline src/js/msp/MSPHelper.js:2036]: Good find. Made a PR changing this single line as RC2 is waiting.

File: src/js/tabs/pid_tuning.js
Comment (qvasic) [reply src/js/tabs/pid_tuning.js]: Done.

File: src/js/tabs/pid_tuning.js
Comment (qvasic) [reply src/js/tabs/pid_tuning.js]: Done.

**Review by haslinghuis — APPROVED** [review]

**Comment by blckmn** [comment]
AUTOMERGE: (FAIL)
 - github identifies PR as mergeable -> FAIL
 - assigned to a milestone -> PASS
 - cooling off period lapsed -> FAIL
 - commit count less or equal to three -> PASS
 - `Don't merge` label NOT found -> PASS
 - at least one `RN:` label found -> PASS
 - `Tested` label found -> PASS
 - assigned to an approver -> FAIL
 - approver count at least three -> FAIL

**Review by KarateBrot — APPROVED** [review]

**Pull request merged**
